src/components/PurchaseOrder/POInvoices/POInvoices.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import {
+  find,
+  get,
+} from 'lodash';
+
+import { MultiColumnList } from '@folio/stripes/components';
+import { AmountWithCurrencyField } from '@folio/stripes-acq-components';
+
+import FolioFormattedDate from '../../FolioFormattedDate';
+
+const POInvoices = ({ orderInvoices, vendors }) => {
+  const resultFormatter = {
+    'invoice': invoice => (
+      <Link
+        data-test-link-to-invoice
+        to={`/invoices/view/${invoice.id}`}
+      >
+        {get(invoice, 'folioInvoiceNo', '')}
+      </Link>
+    ),
+    'invoiceDate': invoice => <FolioFormattedDate value={get(invoice, 'invoiceDate')} />,
+    'vendorName': invoice => get(find(vendors, ['id', get(invoice, 'vendorId', '')]), 'name', ''),
+    'status': invoice => get(invoice, 'status', ''),
+    'expendedAmount': invoice => (
+      <AmountWithCurrencyField
+        currency={get(invoice, 'currency')}
+        amount={get(invoice, 'total', 0)}
+      />
+    ),
+  };
+
+  return (
+    <MultiColumnList
+      contentData={orderInvoices}
+      formatter={resultFormatter}
+      visibleColumns={['invoice', 'invoiceDate', 'vendorName', 'status', 'expendedAmount']}
+      columnMapping={{
+        invoice: <FormattedMessage id="ui-orders.relatedInvoices.invoice" />,
+        invoiceDate: <FormattedMessage id="ui-orders.relatedInvoices.invoiceDate" />,
+        vendorName: <FormattedMessage id="ui-orders.relatedInvoices.vendorName" />,
+        status: <FormattedMessage id="ui-orders.relatedInvoices.status" />,
+        expendedAmount: <FormattedMessage id="ui-orders.relatedInvoices.expendedAmount" />,
+      }}
+      columnWidths={{
+        invoice: '20%',
+        invoiceDate: '15%',
+        vendorName: '25%',
+        status: '15%',
+        expendedAmount: '25%',
+      }}
+    />
+  );
+};
+
+POInvoices.propTypes = {
+  orderInvoices: PropTypes.arrayOf(PropTypes.object),
+  vendors: PropTypes.arrayOf(PropTypes.object),
+};
+
+POInvoices.defaultProps = {
+  orderInvoices: [],
+  vendors: [],
+};
+
+POInvoices.displayName = 'POInvoices';
```

Comment:
https://reactjs.org/docs/react-component.html#displayname
have you checked if this property is required in debugging? according to docs, it should be fine without that

> The displayName string is used in debugging messages. Usually, you don’t need to set it explicitly because it’s inferred from the name of the function or class that defines the component. You might want to set it explicitly if you want to display a different name for debugging purposes or when you create a higher-order component

Reply:
It works fine. thanks.